src/backend/core/models.py

```diff
@@ -213,21 +205,25 @@ def get_abilities(self, user):
         Compute and return abilities for a given user on the template.
         """
         # Compute user role
-        role = None
+        roles = []
         if user.is_authenticated:
             try:
-                role = self.user_role
+                roles = self.user_roles or []
             except AttributeError:
+                teams = user.get_teams()
                 try:
-                    role = self.accesses.filter(user=user).values("role")[0]["role"]
+                    roles = self.accesses.filter(
+                        models.Q(user=user) | models.Q(team__in=teams)
+                    ).values_list("role", flat=True)
```

Comment:
hmm after trying : no because I can't do it on the queryset side for the case when it is precalculated. It really doesn't seem like a problem as the most you can have is a few roles on a given team...

Reply:
Let's merge it :wine_glass: